samples/azure-devops-ci-cd-spfx/AzureDevOps/build/buildDefinition.yaml

```diff
@@ -0,0 +1,90 @@
+resources:
+- repo: self
+queue:
+  name: Hosted VS2017
```

Comment:
just out of curiosity, have you tried it on other agents (like linux for example) ?

Reply:
Npo I didn't, but I could just run them and try it. Probably that would only work with build tasks, that are programmed with nodejs.